compiler/rustc_passes/src/stability.rs

```diff
@@ -144,7 +144,7 @@ impl<'a, 'tcx> Annotator<'a, 'tcx> {
         }
 
         if !self.tcx.features().staged_api {
-            // Propagate unstability.  This can happen even for non-staged-api crates in case
+            // Propagate instability.  This can happen even for non-staged-api crates in case
```

Comment:
I think this should stay "unstability" -- it's technically not a word, but the attribute is called `unstable`.